lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -124,6 +127,36 @@ export class BrowserCacheManager extends CacheManager {
         }
     }
 
+    /**
+     * Searches all cache entries for MSAL accounts and creates the account key map
+     * This is used to migrate users from older versions of MSAL which did not create the map.
+     * @returns 
+     */
+    private createAccountKeyMap(): void {
+        const accountKeys = this.getItem(StaticCacheKeys.ACCOUNT_KEYS);
+        if (accountKeys) {
+            // Account key map already exists, no need to iterate through cache
```

Comment:
Please add telemetry so we know whether partners are using this flow vs the legacy scan everything approach.  This will let us pivot perf queries around this for other perf investigations.

Reply:
Just to clear up some confusion on the approach here, it's not an either or. This new flow is always used for cache lookups regardless of the migration setting. The "legacy scan everything approach" is a one time fix up, if enabled and if needed, on initialization.

I'll see what I can do about adding telemetry here. We currently don't instrument anything in the CacheManager and don't instrument the PCA constructor so this may need to be tackled separately